### DISCUSSION TITLE
Images issue

### DISCUSSION THREAD

**Discussion by spionuit** [community/community#46754] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

  Hello,

i have this problem:

1.I make a webpage using Html ans css. I put on my webpage pictures from my pc, in a folder.
I atach all my code + my folder with pictures, but when i want see my webpage i can't see pictures.

2.If i upload my pictures on imageshuck. imgur or another site, also can't see my pictures. Only if a take a link from internet and put in my code.

 How i can solve this?

**Comment by Synceratus** [comment]
Can you share your code? So we can understand exactly what you are trying to do.

**Reply by spionuit** [reply]
i'm new in this, i dont know if i give to you what you want:
https://spionuit.github.io/Iasi2/

https://github.com/spionuit/Iasi2

my uploads:
[Imagini](https://github.com/spionuit/Iasi2/tree/main/Imagini) 
[Add files via upload](https://github.com/spionuit/Iasi2/commit/01a85fc1534b9f1f91a862573797880bdb82d82c) 
December 4, 2022 15:29 
[Biserici.html](https://github.com/spionuit/Iasi2/blob/main/Biserici.html) 
[Add files via upload](https://github.com/spionuit/Iasi2/commit/01a85fc1534b9f1f91a862573797880bdb82d82c) 
December 4, 2022 15:29 
[Facultati.html](https://github.com/spionuit/Iasi2/blob/main/Facultati.html) 
[Add files via upload](https://github.com/spionuit/Iasi2/commit/01a85fc1534b9f1f91a862573797880bdb82d82c) 
December 4, 2022 15:29 
[Informatii.html](https://github.com/spionuit/Iasi2/blob/main/Informatii.html) 
[Add files via upload](https://github.com/spionuit/Iasi2/commit/01a85fc1534b9f1f91a862573797880bdb82d82c) 
December 4, 2022 15:29 
[Institutii.html](https://github.com/spionuit/Iasi2/blob/main/Institutii.html) 
[Add files via upload](https://github.com/spionuit/Iasi2/commit/01a85fc1534b9f1f91a862573797880bdb82d82c) 
December 4, 2022 15:29 
[Muzee.html](https://github.com/spionuit/Iasi2/blob/main/Muzee.html) 
[Add files via upload](https://github.com/spionuit/Iasi2/commit/01a85fc1534b9f1f91a862573797880bdb82d82c) 
December 4, 2022 15:29 
[Parcuri.html](https://github.com/spionuit/Iasi2/blob/main/Parcuri.html) 
[Add files via upload](https://github.com/spionuit/Iasi2/commit/01a85fc1534b9f1f91a862573797880bdb82d82c) 
December 4, 2022 15:29 
[Teatre.html](https://github.com/spionuit/Iasi2/blob/main/Teatre.html) 
[Add files via upload](https://github.com/spionuit/Iasi2/commit/01a85fc1534b9f1f91a862573797880bdb82d82c) 
December 4, 2022 15:29 
[index.html](https://github.com/spionuit/Iasi2/blob/main/index.html) 
[Add files via upload](https://github.com/spionuit/Iasi2/commit/01a85fc1534b9f1f91a862573797880bdb82d82c) 
December 4, 2022 15:29 
[style.css](https://github.com/spionuit/Iasi2/blob/main/style.css) 
[Add files via upload](https://github.com/spionuit/Iasi2/commit/01a85fc1534b9f1f91a862573797880bdb82d82c)

**Reply by spionuit** [reply]
or like this:
<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Muzee din Iasi</title>
    <link rel="stylesheet" href="style.css" />
    <style>
        body {
            background-color: burlywood;
        }
    </style>
</head>
<body>
    <a href="index.html">Inapoi la Pagina principala</a>
    
</body>
    <h1> LISTA MUZEELOR IASI</h1>
    <ul>
        <li>
            <h2>Bojdeuca lui Ion Creangă</h2>
            <img src="/Site 3 Iasi/Imagini/Bojdeuca-lui-Ion-Creangă.webp" width="200" height="200" align="left" alt="">
            <p>fdgdfgdf
        </li>
    </ul>
    <br>
    <br>
    <hr class="linie">
    <ul>
        <li>
            <h2>Casa Memorială „Mihail Sadoveanu” din Iași</h2>
            <img src="/Site 3 Iasi/Imagini/Casa-Memorială-„Mihail-Sadoveanu”-din-Iași.jpg" width="200" height="160" align="left" alt="">
            <P>fghgfh</p>
        </li>
    </ul>
    <hr>
    <ul>
        <li>
            <h2>Muzeul Municipal din Iași</h2>
            <img src="/Site 3 Iasi/Imagini/Muzeul-Municipal-din-Iași.jpg" width="200" height="200" align="left" alt="">
            <p>ghjgh.</p>
        </li>
    </ul>
</body>


</html>

**Reply by spionuit** [reply]
> Can you share your code? So we can understand exactly what you are trying to do.

can you help me?

**Comment by radar-blip** [comment]
I think you have links messed up a little.

try changing this:
`background-image:url("/Site\ 3\ Iasi/Imagini/Poza2.jpg");`

to this:
`background-image:url("Imagini/Poza2.jpg");`

If this works, repeat in all your html files.

**Comment by spionuit** [comment]
<div class="footer">
          <img src="/Imagini/LOGO SSS(1).png" with="100px" height="100px">
          <span style="padding-left: 100px;">
        </span>

or just using <img src="/Imagini/LOGO SSS(1).png">

if a put a imagine directly from my folder, after i copy on github, i can;t see my picturea.
If i use a picture from internet, is no problem.

How i can upload my pictures to github to see them in my webpage?
      </div>

**Reply by spionuit** [reply]
......<div class="footer">
          <img src="/Imagini/LOGO SSS(1).png" with="100px" height="100px">
          <span style="padding-left: 100px;">
          <h2>&copy;Smart Software Solutions (smart_software_sol@yahoo.com)</h2>;
        </span>
      ....</div>................
